notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<blockquote>a new target, <strong>Draco II</strong> and compare it with <strong>Leo I</strong>.  </blockquote><p>a new target, <strong>Draco II, </strong>which is a UFD galaxy<strong> </strong>and compare it with <strong>Leo I, </strong>a classical dwarf spheroidal.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>